registry/storage/registry.go

```diff
@@ -175,7 +175,8 @@ func (repo *repository) Manifests(ctx context.Context, options ...distribution.M
 
 				// TODO(stevvooe): linkPath limits this blob store to only
 				// manifests. This instance cannot be used for blob checks.
-				linkPathFns: manifestLinkPathFns,
+				linkPathFns:            manifestLinkPathFns,
+				resumableDigestEnabled: repo.resumableDigestEnabled,
```

Comment:
this one isn't strictly necessary since the manifest store doesn't use a blobWriter.